### DISCUSSION TITLE
Text Alignment Issue in gtkwebkit: Text Not Vertically Centered

### DISCUSSION THREAD

**Discussion by greenhat616** [tauri-apps/tauri#10827] [Q&A]
**Description:**

I've noticed a misalignment issue with the text rendering in `gtkwebkit`. Currently, the text alignment appears to be at the top, which deviates from the expected design where the text should be vertically centered.

**Screenshot:**

![Screenshot showing the misaligned text in gtkwebkit](https://github.com/user-attachments/assets/d2dba19d-8c4c-4846-82dc-4c2b32e1af4c)

**Expected Behavior:**

The text should be vertically centered within its container, maintaining consistent alignment across all platforms.

**Additional Information:**

This issue does not occur in `webview2` on Windows or in the webview component on macOS, where the text alignment behaves as expected. 

Could we investigate why this inconsistency is happening specifically with `gtkwebkit`?

**Comment by FabianLars** [comment]
> Could we investigate why this inconsistency is happening specifically with gtkwebkit?

We, as in the Tauri Team, do not have any capacity to help with that. We'd appreciate it if you could reach out to the [webkitgtk](https://webkitgtk.org/) team though!
